docs/technical/shell-integration.md

```diff
@@ -172,21 +172,21 @@ use Hyper as a reference to explain the rest of the process.
 
 ### Step 1: Find the shell application
 
-The `getBundleID()` function is used to map a shell enum to it's bundle ID
+The `getBundleID()s` function is used to map a shell enum to it's bundle ID
 that is defined in it's manifest. You should add a new entry here for your
```

Comment:
This is not your typo but it'd be nice to fix it now that you have the chance 😅 
```suggestion
that is defined in its manifest. You should add a new entry here for your
```

Reply:
Fixed